SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -100,9 +100,9 @@ private static void PlatformNotifyUserConfirmation(UserConfirmation userConfirma
             Instance.InstanceHandlerUserConfirmation(userConfirmation);
         }
 
-        private static void PlatformTrackError(System.Exception exception, IDictionary<string, string> properties, ErrorAttachmentLog[] attachments)
+        private static Guid PlatformTrackError(System.Exception exception, IDictionary<string, string> properties, ErrorAttachmentLog[] attachments)
```

Comment:
I see that only `private` methods were modified, so it's not clear how exactly this will solve the initial problem.

Probably, `SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Shared/Crashes.cs` should be modified as well.

Reply:
pushed a newer commit, with those changes and also replaced GUID to be nullable to return null instead of guid.empty incase its inactive